demo/App.js

```diff
@@ -1,8 +1,8 @@
-import './styles/global';
-import React from 'react';
-import { View, Text } from 'react-native';
+import "./styles/global";
```

Comment:
can we maintain the single quote?

Reply:
With the prettier config, eslint is throwing errors for single quotes. Passing the --fix flag to eslint automatically fixed the above changes.

Personally I haven't come across many js projects using single quotes. I suggest we go with double quotes.

Reply:
That's weird because if you see the conf file for prettier it explicitly says to use single quotes: https://github.com/rootstrap/react-native-use-styles/blob/2dab777b37804b9a7e3e5f2d31e355832a40f70a/.pretierrc#L2